### PR TITLE
Count private clients in current clients

### PR DESCRIPTION
**Pull request by Avygeil**

### CODE REVIEW

File: codemp/server/sv_main.cpp [hunk 1]
```diff
@@ -598,8 +598,7 @@ void SVC_Info( netadr_t from ) {
 	Info_SetValueForKey( infostring, "mapname", sv_mapname->string );
 	Info_SetValueForKey( infostring, "clients", va("%i", count) );
 	Info_SetValueForKey( infostring, "g_humanplayers", va("%i", humans) );
-	Info_SetValueForKey( infostring, "sv_maxclients",
-		va("%i", sv_maxclients->integer - sv_privateClients->integer ) );
+	Info_SetValueForKey( infostring, "sv_maxclients", va("%i", sv_maxclients->integer ) );
```
Comment (dpadgett) [inline codemp/server/sv_main.cpp:601]: why?  with this it will look like there is still space on a full server, people will try to connect and get booted

Comment (deathsythe47) [inline codemp/server/sv_main.cpp:601]: We should keep max clients the same, but just bump up the number of clients to include the private users. That way you can see how many people are on the server. Thus a full 14 player server with 2 private clients will show as 16/14.

Comment (Avygeil) [inline codemp/server/sv_main.cpp:601]: stop trying to get credits for my ideas fag

Comment (Avygeil) [inline codemp/server/sv_main.cpp:601]: Well it is still ambiguous, for example if there are 12 clients and 2 private clients, server would show 14/14 even though there is still space. This should stay a client side fix imo (14/14+2).

Comment (dpadgett) [inline codemp/server/sv_main.cpp:601]: true yeah.  feels to me like the status quo is the best option for now.

**Pull request closed without merging**